requirements-dev.txt

```diff
@@ -28,7 +28,7 @@ requests
 scikit-image
 scipy
 sphinx
-sphinx_rtd_theme
+sphinx_rtd_theme==0.5.2
```

Comment:
Please open an issue reminding us to follow up on this.

Reply:
I am afraid `sphinx_rtd_theme` needs to be pinned in all projects. The new version v1.0.0 does not work for some reason. So this is a general issue.